lib/portage/package/ebuild/prepare_build_dirs.py

```diff
@@ -477,13 +477,21 @@ def _ensure_log_subdirs(logdir, subdir):
         ensure_dirs(current, uid=uid, gid=gid, mode=grp_mode, mask=0)
 
 
+def _safe_copytree(src, dst, **kwargs):
```

Comment:
That's a weird name for a function that removes `dst` without warning.

Reply:
Good point. Renamed to just `_copytree`.